docs/telemetry.rst

```diff
@@ -232,3 +233,51 @@ Example of recorded documents given two data streams in the cluster::
 Supported telemetry parameters:
 
 * ``data-stream-stats-sample-interval`` (default 10): A positive number greater than zero denoting the sampling interval in seconds.
+
+ingest-pipeline-stats
+---------------------
+
+The ingest-pipeline-stats telemetry device makes a call at the beginning and end of the benchmark to the `node stats API (_nodes/stats/ingest) <https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-nodes-stats.html>`_ and records the deltas in the form of:
+
+  * Three results documents for each cluster: ``ingest_pipeline_cluster_count``, ``ingest_pipeline_cluster_time``, ``ingest_pipeline_cluster_failed``
```

Comment:
Do we want to show this in the final summary?

Reply:
I'm not sure I follow? Do you mean the final race report? If so, these are the stats that are reported already.

Reply:
Sorry, I found out they were in there after and forgot to remove the comment.